### ISSUE TITLE
Php7 and incorrect types error.

### ISSUE THREAD

**Issue by akaSStalkALEX** [bedezign/yii2-audit#141]
```
Uncaught TypeError: Argument 2 passed to bedezign\yii2\audit\panels\ErrorPanel::log() must be an instance of Exception, instance of Error given, called in /vagrant/vendor/bedezign/yii2-audit/src/components/base/ErrorHandlerTrait.php on line 42 and defined in /vagrant/vendor/bedezign/yii2-audit/src/panels/ErrorPanel.php:46
Stack trace:
#0 /vagrant/vendor/bedezign/yii2-audit/src/components/base/ErrorHandlerTrait.php(42): bedezign\yii2\audit\panels\ErrorPanel->log(1426, Object(Error))
#1 /vagrant/vendor/yiisoft/yii2/base/ErrorHandler.php(105): bedezign\yii2\audit\components\web\ErrorHandler->logException(Object(Error))
#2 [internal function]: yii\base\ErrorHandler->handleException(Object(Error))
#3 {main}
thrown
```

**Comment by krists1** [comment]
Hi,
I found the problem. In class bedezign\yii2\audit\panels\ErrorPanel

    /**
     * Log an exception
     *
     * @param int        $entry_id      Entry to associate the error with
     * @param Exception  $exception
     * @return null|static
     */
    public function log($entry_id, Exception $exception)
    {
      ...
    }

The function expects an exception, but in case of fatal error (i.e. ParseError) php7 hits it with Throwable. Just remove the Exception in function definition and everything works as expected.
Hope it helps!

**Comment by akaSStalkALEX** [comment]
Yes, sure that is a problem I meant. I thought maybe it should be fixed, this package presents in composer though.

**Comment by Blizzke** [comment]
I'll take care of this tonight when at home. Thank you for reporting it.

**Comment by Blizzke** [comment]
Fixed in [1.0.6](https://github.com/bedezign/yii2-audit/releases/tag/1.0.6)

**Comment by akaSStalkALEX** [comment]
Thank you :)
